CHANGELOG.md

```diff
@@ -25,7 +42,7 @@
 * **toolkit/observable:** Support for disabling the native `ResizeObserver` has been removed because this API is now supported by all major browsers.
 
   To migrate:
-  - remove the `options` parameter when calling `fromDimension$`.
+  - remove the `options` parameter when calling `fromDimension$ `.
```

Comment:
`markdown-include` fails to embed the changelog if encountering the construct "$`". For this reason, I have appended a space, as we have done in previous commits.